pom.xml

```diff
@@ -1074,6 +1093,27 @@
 					</execution>
 				</executions>
 			</plugin>
+
+			<!-- Generates the OpenAPI rest service definition file during build time -->
+			<plugin>
+				<groupId>org.springdoc</groupId>
+				<artifactId>springdoc-openapi-maven-plugin</artifactId>
+				<version>${org.springdoc.openapi.maven.version}</version>
+				<executions>
+					<execution>
+						<id>integration-test</id>
+						<goals>
+							<goal>generate</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<apiDocsUrl>http://localhost:${jetty.port}/v3/api-docs</apiDocsUrl>
+					<outputFileName>open-api.json</outputFileName>
+					<outputDir>doc/swagger-ui</outputDir>
```

Comment:
Can you add an entry to the `.gitignore` for this folder?  That way it won't show up as a new directory all the time after generating the docs.

Reply:
Of course! Please see d185de210af77eba99c2943aad4456db51eae787.